features/support/service_helper.rb

```diff
@@ -36,20 +36,42 @@ def upload_extra_data_set(service)
     run_all_delayed_jobs
   end
 
-  def create_service(name)
+  def create_service(params)
     mapit_knows_nothing_about_any_postcodes
 
+    params.reverse_merge!(service_defaults)
```

Comment:
Prefer `params = service_defaults.merge(params)` to avoid mutating the object we were given (and me having to remember what `reverse_merge` does).